tools/ci/Dockerfile

```diff
@@ -1,29 +1,32 @@
-FROM ubuntu:18.04
-
+FROM ubuntu:18.04 as builder
 LABEL maintainer service@fisco.com.cn
 
-ARG SOURCE_BRANCH
-ENV SOURCE=${SOURCE_BRANCH:-master}
-
 WORKDIR /
 
+ARG SOURCE_BRANCH
+ENV DEBIAN_FRONTEND=noninteractive \
+    SOURCE=${SOURCE_BRANCH:-master}
+
 RUN apt-get -q update && apt-get install -qy --no-install-recommends \
     git clang make build-essential cmake libssl-dev zlib1g-dev ca-certificates \
-    && export DEBIAN_FRONTEND=noninteractive \
     && ln -fs /usr/share/zoneinfo/Asia/Shanghai /etc/localtime \
     && apt-get install -qy --no-install-recommends tzdata \
-    && dpkg-reconfigure --frontend noninteractive tzdata \
-    && git clone https://github.com/FISCO-BCOS/FISCO-BCOS.git \
+    && dpkg-reconfigure --frontend noninteractive tzdata
+RUN git clone https://github.com/FISCO-BCOS/FISCO-BCOS.git \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use WORKDIR to switch to a directory](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5318653)